test_scripts/API/PerformAudioPassThru/017_ATF_N_PerformAudioPassThru_INVALID_DATA_All_Params_audioPassThruIcon_invalid_ImageType.lua

```diff
@@ -0,0 +1,110 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PerformAudioPassThru] SDL must transfer request to HMI in case of valid "audioPassThruIcon" param
+-- [GeneralResultCodes] INVALID_DATA wrong type
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case that the request comes with
+-- wrong type (empty) parameter(including parameters of the structures)
+-- SDL must respond with resultCode "INVALID_DATA" and success:"false"
+--
+-- 1. Used preconditions
+-- 1.1. Request is sent with audioPassThruIcon image structure with invalid ImageType parameter
+-- and valid value parameter, all other params used in PerformAudioPassThru_request
+-- are present and within bounds
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon (valid value parameter,ImageType is not valid), other params) from mobile to SDL and check:
+--
+-- Expected result:
+-- SDL sends PerformAudioPassThru (INVALID_DATA, success:false) to mobile app
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPostconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ Local variables ]]
+local params_invalid_data =
+{  
+  {param_value = "", comment = "ImageType_is_empty"}, -- audioPassThruIcon with empty ImageType parameter
+  {param_value = 123, comment = "ImageType_wrong_type"}, --audioPassThruIcon contains wrong type (not predefined in enum STATIC/DYNAMIC) in ImageType parameter
```

Comment:
@maria-mih, will be better to comment "ImageType_wrong_type" like "ImageType_integer_type".
Also update comment "(not predefined in enum STATIC/DYNAMIC)" - the comment should be for integer value.